doc/references.bib

```diff
@@ -227,6 +227,23 @@ @article{ColcloughEtAl2015
   pages = {439--448}
 }
 
+@article{RecanatesiEtAl2022,
```

Comment:
Add:

```
@article{LiAdam2021,
   abstract = {Over 15 million patients with epilepsy worldwide do not respond to drugs. Successful surgical treatment requires complete removal or disconnection of the seizure onset zone (SOZ), brain region(s) where seizures originate. Unfortunately, surgical success rates vary between 30 and 70% because no clinically validated biological marker of the SOZ exists. We develop and retrospectively validate a new electroencephalogram (EEG) marker—neural fragility—in a retrospective analysis of 91 patients by using neural fragility of the annotated SOZ as a metric to predict surgical outcomes. Fragility predicts 43 out of 47 surgical failures, with an overall prediction accuracy of 76% compared with the accuracy of clinicians at 48% (successful outcomes). In failed outcomes, we identify fragile regions that were untreated. When compared to 20 EEG features proposed as SOZ markers, fragility outperformed in predictive power and interpretability, which suggests neural fragility as an EEG biomarker of the SOZ.},
   author = {Adam Li and Chester Huynh and Zachary Fitzgerald and Iahn Cajigas and Damian Brusko and Jonathan Jagid and Angel O. Claudio and Andres M. Kanner and Jennifer Hopp and Stephanie Chen and Jennifer Haagensen and Emily Johnson and William Anderson and Nathan Crone and Sara Inati and Kareem A. Zaghloul and Juan Bulacio and Jorge Gonzalez-Martinez and Sridevi V. Sarma},
   doi = {10.1038/s41593-021-00901-w},
   issn = {15461726},
   issue = {10},
   journal = {Nature Neuroscience},
   title = {Neural fragility as an EEG marker of the seizure onset zone},
   volume = {24},
   year = {2021},
}
```

Reply:
to be consistent with MNE-Python bibtex conventions, the entry key should be `LiEtAl2021` and probably no need to include the abstract (it will never get displayed on the website).

Also at least in the MNE-Python repo, `doc/references.bib` is alphabetized by first author (except for the first 2 entries, which are the historical "canonical citations")